content/rendering-with-batch/_index.md

```diff
@@ -11,7 +11,7 @@ The estimated completion time of this lab is **90 minutes**. Please note that re
 {{% /notice %}}
 ## Overview
 
-In this workshop you will learn to submit jobs with [AWS Batch](https://aws.amazon.com/batch/) following Spot best practices to [render](https://en.wikipedia.org/wiki/Rendering_(computer_graphics)) a [Blender](https://www.blender.org/) file in a distributed way. You will be creating a docker container and publishing it in Amazon Elastic Container Registry (ECR). Then you will use that container in AWS Batch using a mix of EC2 On-Demand and Spot instances. [Spot instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) are EC2 spare capacity offered at steep discounts compared to On-Demand instances and are a cost-effective choice for applications that can be interrupted, what makes them well-suited for the batch processing that we will run. After going through all the sections, you will have the following pipeline created, orchestrated by AWS Step Functions:
+In this workshop you will learn to submit fault tolerant jobs with [AWS Batch](https://aws.amazon.com/batch/) following Spot best practices to [render](https://en.wikipedia.org/wiki/Rendering_(computer_graphics)) a [Blender](https://www.blender.org/) file in a distributed way. You will be creating a docker container and publishing it in Amazon Elastic Container Registry (ECR). Then you will use that container in AWS Batch using a mix of EC2 On-Demand and Spot Instances. AWS Batch workloads are usually a perfect fit for [Spot Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html), EC2 spare capacity offered at steep discounts compared to On-Demand instances. Using [AWS Fault Injection Simulator(FIS)](https://aws.amazon.com/fis/), we will simulate an EC2 [Spot Instance interruption](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-interruptions.html). AWS Batch will automatically retry the task and spin-up another Spot Instance using AWS EC2 Spot Best practices. After going through all the sections, you will have a resilient pipeline created, orchestrated by AWS Step Functions:
```

Comment:
Please be careful about writing in first or third person. If the whole paragraph was already written in third person don't mix it ("we will simulate....")

If the name of the service is AWS Fault Injection Simulator, the shortened version is AWS FIS, not FIS. Correct please and leave a blank space before the parenthesis.

Remove "AWS Batch will automatically retry the task and spin-up another Spot Instance using AWS EC2 Spot Best practices" too specific for the workshop intro